sbpy/photometry/core.py

```diff
@@ -1026,6 +1026,8 @@ def _G2(self):
 
     @staticmethod
     def evaluate(ph, h, g1, g2):
+        if isinstance(ph, u.Quantity):
+            ph = ph.to('rad').value
```

Comment:
Where is the default behavior (assumed units on `ph`) documented?  Would `ph = u.Quantity(ph, 'deg').to_value('rad')` be acceptable?  That way the default (if it is degrees) is clear in the code and the if statement can be avoided.

Reply:
Good trick.  Will do it.  Thanks.